Analyzer/include/Config.h

```diff
@@ -60,21 +61,21 @@ class Config
             else if(module=="MegaJetCombine")               tr.emplaceModule<MegaJetCombine>();
             else if(module=="FatJetCombine")                tr.emplaceModule<FatJetCombine>();
             else if(module=="TrainingNTupleVars")          tr.emplaceModule<TrainingNTupleVars>();
-            else if(module=="MakeStopHemispheres_All")      tr.emplaceModule<MakeStopHemispheres>("Jets",     "AllJets",       "NJets",          "_All");
-            else if(module=="MakeStopHemispheres_1l")       tr.emplaceModule<MakeStopHemispheres>("Jets",     "GoodJets_pt30", "NGoodJets_pt30", "_1l");
-            else if(module=="MakeStopHemispheres_0l")       tr.emplaceModule<MakeStopHemispheres>("Jets",     "GoodJets_pt45", "NGoodJets_pt45", "_0l");
-            else if(module=="MakeStopHemispheres_TaggedTop") tr.emplaceModule<MakeStopHemispheres>("StopJets", "GoodStopJets",  "NGoodStopJets",  "_TaggedTop");
+            else if(module=="MakeStopHemispheres_All")      tr.emplaceModule<MakeStopHemispheres>("Jets",     "AllJets",       "NJets",          "_All", 2);
+            else if(module=="MakeStopHemispheres_1l")       tr.emplaceModule<MakeStopHemispheres>("Jets",     "GoodJets_pt30", "NGoodJets_pt30", "_1l", 2);
+            else if(module=="MakeStopHemispheres_0l")       tr.emplaceModule<MakeStopHemispheres>("Jets",     "GoodJets_pt20", "NGoodJets_pt20", "_0l", 2);
+            else if(module=="MakeStopHemispheres_TaggedTop") tr.emplaceModule<MakeStopHemispheres>("StopJets", "GoodStopJets",  "NGoodStopJets",  "_TaggedTop", 5);
```

Comment:
I like that you are passing the number 5 now, but it would be better to have a enum defined in the MakeStopHemishperes classs.